docs/_plugins/3rd-party-integration/wp-mapit.md

```diff
@@ -1,9 +1,9 @@
 ---
 name: WP MapIt
 category: 3rd-party-integration
-repo: http://wp-mapit.phpwebdev.in/
+repo: http://wp-mapit.chandnipatel.in/
```

Comment:
Why not [https](https://wp-mapit.chandnipatel.in/)?

Reply:
Done

Reply:
https://github.com/Leaflet/Leaflet/pull/7986